layer/executables/packager.py

```diff
@@ -105,6 +109,10 @@ def get_function_package_info(package_path: Path) -> FunctionPackageInfo:
     )
 
 
+def _is_dev_version() -> Boolean:
+    return layer.__version__ == "0.10.0b1"
```

Comment:
Would `__layer.__version__.endswith("0b1")` be a more stable heuristic?

Reply:
I wanted this to be as strict as possible. If we ever decide to publish any beta versions, we don't want to accidentally trigger this logic.

If we change the local version, the test will fail and we can update this constant as well.